torch/csrc/jit/pybind_utils.h

```diff
@@ -220,9 +218,6 @@ inline IValue toIValue(
     case TypeKind::ProfiledTensorType:
     case TypeKind::CompleteTensorType: {
       auto var = py::cast<autograd::Variable>(obj);
-      if (var.is_sparse()) {
```

Comment:
We can't delete these without more work to figure out what is needed to support sparse tensors in the JIT.

Reply:
reverted jit changes and excluded to_sparse from jit tests instead.

Is there a plan to support sparse tensors in jit at some point?